freqtrade/exchange/exchange.py

```diff
@@ -1270,40 +1270,64 @@ def extract_cost_curr_rate(self, order: Dict) -> Tuple[float, str, Optional[floa
 
     # Historic data
 
-    def get_historic_ohlcv(self, pair: str, timeframe: str,
-                           since_ms: int, is_new_pair: bool = False) -> List:
+    def get_historic_ohlcv(
+        self,
+        pair: str,
+        timeframe: str,
+        since_ms: int,
+        is_new_pair: bool = False,
+        price: Optional[str] = None
+    ) -> List:
         """
         Get candle history using asyncio and returns the list of candles.
         Handles all async work for this.
         Async over one pair, assuming we get `self.ohlcv_candle_limit()` candles per call.
         :param pair: Pair to download
         :param timeframe: Timeframe to get data for
         :param since_ms: Timestamp in milliseconds to get history from
+        :param price: "mark" if retrieving the mark price cnadles, "index" for index price candles
         :return: List with candle (OHLCV) data
         """
         return asyncio.get_event_loop().run_until_complete(
-            self._async_get_historic_ohlcv(pair=pair, timeframe=timeframe,
-                                           since_ms=since_ms, is_new_pair=is_new_pair))
+            self._async_get_historic_ohlcv(
+                pair=pair,
+                timeframe=timeframe,
+                since_ms=since_ms,
+                is_new_pair=is_new_pair,
+                price=price
+            ))
 
-    def get_historic_ohlcv_as_df(self, pair: str, timeframe: str,
-                                 since_ms: int) -> DataFrame:
+    def get_historic_ohlcv_as_df(
+        self,
+        pair: str,
+        timeframe: str,
+        since_ms: int,
+        price: Optional[str] = None
+    ) -> DataFrame:
         """
         Minimal wrapper around get_historic_ohlcv - converting the result into a dataframe
         :param pair: Pair to download
         :param timeframe: Timeframe to get data for
         :param since_ms: Timestamp in milliseconds to get history from
+        :param price: "mark" if retrieving the mark price cnadles, "index" for index price candles
```

Comment:
```suggestion
        :param price: "mark" if retrieving the mark price candles, "index" for index price candles
```

same typo is basically in all docstrings you added here ... :laughing: